src/main/scala/com/mozilla/telemetry/streaming/FederatedLearningSearchOptimizer.scala

```diff
@@ -36,6 +36,7 @@ object FederatedLearningSearchOptimizer extends StreamingJobBase {
       .option("spark.streaming.kafka.consumer.cache.maxCapacity", 100)
       .option("subscribe", TelemetryKafkaTopic)
       .option("startingOffsets", opts.startingOffsets())
+      .option("failOnDataLoss", false)
```

Comment:
Can we instead use configuration option here to keep this consistent with other jobs?
You'd need to change this like:
https://github.com/mozilla/telemetry-streaming/blob/3c0a0015f91287b43743f0f2bae3b1b7d4addbde/src/main/scala/com/mozilla/telemetry/streaming/EventsToAmplitude.scala#L218
and add the option in this class' Opts` declaration, like:
https://github.com/mozilla/telemetry-streaming/blob/3c0a0015f91287b43743f0f2bae3b1b7d4addbde/src/main/scala/com/mozilla/telemetry/streaming/EventsToAmplitude.scala#L81-L82

Also: please [use the imperative mood in the commit message](https://chris.beams.io/posts/git-commit/#imperative)

Reply:
Thanks @akkomar.  I've added d2e827f to add an option that defaults to false for this.